.travis.yml

```diff
@@ -2,7 +2,6 @@ language: node_js
 node_js:
   - "12"
   - "10"
-  - "8"
```

Comment:
To be very explicit: no PR with this change can land, as long as I have a say.

Reply:
Then it's nothing is going to merge. Because without it is failing and master is failing.

Take it or leave it, your attitude is not one that inspires me to contribute further.

Reply:
Since [node v8 is end of life now](https://nodejs.org/en/about/releases/), can we merge this? This would be a great improvement for this tool IMO.

Reply:
A node version being EOL is irrelevant; package authors can and should support engine versions long after their vendors have stopped doing so.

Reply:
> package authors can and should support engine versions long after their vendors have stopped doing so

Ok, I guess. This seems like a very strange hill to die on, though. I think not alienating contributors and enabling a whole word of new use cases beats sacrificing your software on the alter of backwards compatibility. :man_shrugging:

Reply:
For our team, this contribution would have made `install-peerdeps` go from an unusable tool to something really useful!

Thanks for the contribution @amclin! We're looking forward to the day this can be released 😄

Reply:
@ljharb since you refuse to accept a backwards-breaking change, the responsibility is on you to fix master, so that contributions can be made.

Right now, no-one can contribute to this project because master is failing.

This PR is now 9 months old, because master is failing.

Even your automated security patches from Dependabot are not merging, because master is failing.

Fix master, and I'm happy to update this PR to reflect whatever changes you make to master. But no-one else is going to fix master for you, especially since you've made it clear you won't accept others' logical approaches to doing so.

Reply:
It's totally fair to hold off updating this PR until master fixed, but if you had restored node 8 to the testing matrix, and the only failures on this PR were the same ones that were already in master, then I'd consider your PR ready to merge. This is a useful change, and I'd prefer to see it go in without disrupting existing users, and without needing to debate support philosophies.

083177892cd4bdb86f088e2e62b73d302b4f991b should have fixed master, so I've rebased this PR (and restored node 8).